articles/overview/understanding-quantum-computing.md

```diff
@@ -0,0 +1,64 @@
+
+A quantum particle such as an electron has its own “facing left or facing right” property called **spin**, which can be either up or down (or positive or negative for a photon), or to make it more relatable to classical binary computing, let’s just say 1 or 0. When a quantum particle is in a superposition state, it’s a linear combination of an infinite number of states between 1 and 0, but you don’t know which one it will be until you actually look at it, which brings up our next phenomenon, **quantum measurement**.
+
+### Quantum measurement
+
+Now, let’s say your friend comes over and wants to take a picture of you exercising. Most likely, they’ll get a blurry image of you turning somewhere between all the way left and all the way right.
+
+But if you’re a quantum particle, an interesting thing happens. No matter where you are when they take the picture, it will always show you either all the way left or all the way right – nothing in-between.
+
+This is because the act of observing or measuring a quantum particle **collapses** the superposition state (also known as **decoherence**) and the particle takes on a classical binary state of either 1 or 0.
+
+This binary state is helpful to us, because in computing you can do lots of things with 1’s and 0’s. However, once a quantum particle has been measured and collapsed, it stays in that state forever (just like your picture) and will always be a 1 or 0, and cannot be used for more calculations. As you’ll see later, though, in quantum computing there are operations that can “reset” a particle back to a superposition state so it can be useful again.
+
+### Entanglement
+
+Possibly the most interesting phenomenon of quantum physics is the ability of two or more quantum particles to become **entangled** with each other. When particles become entangled, they form a single system such that the quantum state of any one particle cannot be described independently of the quantum state of the other particles. This means that whatever operation or process you apply to one particle correlates to the other particles as well, almost instantaneously.
+
+In addition to this interdependency, particles can maintain this connection even when separated over incredibly large distances, even light-years. The effects of quantum measurement also apply to entangled particles, such that when one particle is measured and collapses, the other particle collapses as well. Because there is a correlation between the entangled qubits, measuring the state of one qubit provides information about the state of the other qubit – this particular property is very helpful in quantum computing. These unique properties of entangled particles can be used to transmit quantum information from one place to another by a process called **teleportation**, and can play a role in some cryptography processes.
```

Comment:
As noted in my previous review, teleportation is not used in most QKD protocols; while Ekert91 is teleportation based, BB84 is prepare-and-measure based.